### PR TITLE
document modelling users and groups in bs

### PR DESCRIPTION
**Pull request by punkle**

### CODE REVIEW

File: content/docs/details/modelling-users-and-groups/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+title: Modelling Users & Groups
+publishedDate: '2022-06-16T16:00:00.0Z'
+description: How to represent users and groups in Backstage and make ownership work.
+---
+
+## Prerequisites
+
+You should already have a knowledge of how to add entities to the Backstage catalog as descibed [here](/docs/getting-started/adding-components/).
```
Comment (kissmikijr) [inline content/docs/details/modelling-users-and-groups/index.md:9]:
```suggestion
You should already have a knowledge of how to add entities to the Backstage catalog as described [here](/docs/getting-started/adding-components/).
```

File: content/docs/integrations/home-page-rss/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    Now the HomepageMarkdown component should be added to your home page.
+
+## Writing the markdown file for the homepage.
+
+There is one important thing in the current version (1.1.1) of the plugin. You'll need to refer to resources (links & images) with absolute urls.
+
+```diff
+- ![](./docs/home-page-markdown.png)
++ ![](https://github.com/RoadieHQ/roadie-backstage-plugins/raw/SC-7064-add-markdown-home-plugin/plugins/home/backstage-plugin-home-markdown/docs/home-page-markdown.png)
```
[the remark this answers is not in the document]
Comment (kissmikijr) [reply content/docs/integrations/home-page-rss/index.md:81]: ah, nah this is needed :D

File: content/docs/integrations/home-page-rss/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+---
```
Comment (kissmikijr) [inline content/docs/integrations/home-page-rss/index.md:1]: This whole file talks about the markdown homepage plugin but it is in the home-page-rss folder, is it an accidental copy paste?

**Review by kissmikijr — CHANGES_REQUESTED** [review]
Nice, one small question about an index.md file

File: content/docs/integrations/home-page-rss/index.md (hunk 3 above)
Comment (punkle) [reply content/docs/integrations/home-page-rss/index.md]: I didnt mean to add this one. Ive removed it

**Review by kissmikijr — APPROVED** [review]

**Pull request merged**